examples/depower_simple_3line.jl

```diff
@@ -0,0 +1,120 @@
+using Pkg, Timers
+tic()
+if ! ("KitePodModels" ∈ keys(Pkg.project().dependencies))
+    using TestEnv; TestEnv.activate()
+end
+
+using KiteViewers, KiteModels, KitePodModels, Rotations
+
+const Model = KPS4_3L
+
+if ! @isdefined kcu;  const kcu = KCU(se());   end
+if ! @isdefined kps4_3l; const kps4_3l = Model(kcu); end
+
+# the following values can be changed to match your interest
+dt::Float64 = 0.3
+const TIME = 50
+const TIME_LAPSE_RATIO = 1
+const STEPS = Int64(round(TIME/dt))
+STATISTIC = true
+SHOW_KITE = true
+PLOT_PERFORMANCE = false
+# USE 3L SETTINGS!
+update_settings()
+# end of user parameter section #
+
+if Model==KPS3 SHOW_KITE = true end
+
+if ! @isdefined time_vec_gc; const time_vec_gc = zeros(STEPS); end
+if ! @isdefined time_vec_sim; const time_vec_sim = zeros(STEPS); end
+if ! @isdefined time_vec_tot; const time_vec_tot = zeros(div(STEPS, TIME_LAPSE_RATIO)); end
+viewer::Viewer3D = Viewer3D(SHOW_KITE)
+
+function simulate(integrator, steps)
+    start = integrator.p.iter
+    start_time_ns = time_ns()
+    j=0; k=0
+    KiteViewers.clear_viewer(viewer)
+    GC.gc()
+    max_time = 0
+    sys_state = SysState(kps4_3l)
+    for i in 1:steps
+        if i == 300
+            # set_depower_steering(kps4_3l.kcu, 0.30, 0.0)
+        elseif i == 640
+            # set_depower_steering(kps4_3l.kcu, 0.35, 0.0)    
+        end
+        t_sim = @elapsed KiteModels.next_step!(kps4_3l, integrator, dt=dt)
```

Comment:
I don't think that this works any longer. Use next_step!(kps4_3l, integrator; set_speed=0, dt=dt) instead.

Reply:
Works, because kps4_3l has a different step! function. https://github.com/1-Bart-1/KiteModels.jl/blob/72583a349db942daedaaafbaa8dc27583a404731/src/KiteModels.jl#L698